doc-content/enterprise-only/ai/ai-gsg-proc.adoc

```diff
@@ -0,0 +1,178 @@
+[id='ai-gsg_{context}']
+
+= Using a PMML file with DMN
+
+Use case: Card transaction dispute
+End user can dispute a card transaction and the system has to decide if the dispute can be automatically processed (aka accept the dispute and refund the user) or if it requires to have manual verification
+
+.Prerequisites:
+
+* {PRODUCT} is available and the following JAR files have been added to the `~/kie-server.war/WEB-INF/lib` and `~/business-central.war/WEB-INF/lib` directories in your {PRODUCT} installation:
++
+** `kie-dmn-jpmml-{BOM_VERSION}.jar`
++
+** `kie-dmn-jpmml-{BOM_VERSION}.jar`
++
+These files are available in the {PRODUCT_DM} {PRODUCT_VERSION}  Maven Repository distribution avaialable from the [https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required).
```

Comment:
Please recheck the link syntax and correct the spelling of available. The rendered output:

These files are available in the Red Hat Decision Manager 7.9 Maven Repository distribution avaialable from the [Software Downloads page in the Red Hat Customer Portal (login required).

Reply:
Fixed